composer.lock

```diff
@@ -58,6 +58,9 @@
                 "doctrine",
                 "orm"
             ],
+            "support": {
```

Comment:
Please remove modifications on composer.lock

Reply:
These are added automatically by composer when you run `composer update nothing`
If I change it then composer.lock will become out of sync with composer.json

Reply:
composer.lock is again out of sync

Reply:
I will update it again once QA has validated it